docs/02-connect/04-eon_block_explorer.md

```diff
@@ -0,0 +1,584 @@
+- **Total transactions** - The total number of transactions in a time period in the blockchain.
+- **Total blocks** - The total number of blocks in the blockchain.
+- **Wallet address** - The total number of wallet addresses associated with transactions in the blockchain.
+
+## Blocks
+
+The Block module provides key information about the block.
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-blocks.png").default} alt="blocks-module" width="900" height="700" />
+</p>
+
+Key information includes:
+
+- **Block number** - The number assigned to the block.
+- **Number of transactions** - The running number of transactions for the block.
+- **Number of bytes** - The amount of bytes for the block.
+- **Elapsed timer for the block** - Clock counting since block creation period.
+- **Miner (wallet) address** - The miner’s address. Click the address to Address Detail page.
+
+## Transactions
+
+The **Transaction** module provides key information about the transaction.
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-transactions.png").default} alt="transactions" width="900" height="700" />
+</p>
+
+Key information includes:
+
+- **Transaction Status** - Success or Error: (Awaiting internal transaction for reason)
+- **Transaction ID (hash)** - The unique identifier that validates the transaction.
+- **Address** - The wallet address associated with the transaction. 
+- **Transaction Fee** - The amount charged to process the transaction.
+- **Contract Address** - The contract address associated with the transaction. Click the address to **Address Detail** page.
+- **(Receiving) Address** - The receiving address associated with the transfer transaction. Click the address receiving asset to **Address Detail** page.
+
+## Gobi Testnet Block Explorer Features
+
+The Gobi Testnet Block Explorer gives you the ability to search the blockchain for transactions, addresses, and statistics, including the customized features of:
+
+- Backward Transfer
+- Forward Transfer
+- Fee Payment
+- Backward Transfer
+
+
+Typically, when a user decides to move ZEN from a sidechain back to the mainchain (to either send ZEN to another sidechain or to cash out, for example) the backward transfer transaction is deployed. Backward transfers are initiated on EON (EVM sidechain) as special transactions batched in withdrawal certificates and propagated by EON nodes to the mainchain.
+
+The Gobi Testnet Explorer highlights the Backward Transfer transaction:
+
+<p>
+<img src={require("/img/docs/blockscout/gobi-exp-backward.png").default} alt="backward-transaction" width="900" height="700" />
+</p>
+
+Key information includes:
+
+- **Transaction Status** - Success or Error: (Awaiting internal transaction for reason)
+- **Transaction ID (hash)** - The unique identifier that validates the transaction. Click to **Transaction Detail** page.
+- **(Sending) Address** - The sending address of the transaction. Click the address receiving asset (link) to **Address Detail** page.

```

Comment:
Edit wording here, similar to previous comment

Reply:
correction made